starfish/spots/_detector/__init__.py

```diff
@@ -71,4 +71,8 @@ def _cli(cls, args, print_help=False):
             blobs_image=blobs_image,
             reference_image_from_max_projection=args.reference_image_from_max_projection
         )
+
+        # When PixelSpotDetector is used run() returns a tuple
```

Comment:
Any way we could adjust this to pickle and save the secondary results that it emits instead of dropping them? A general solution could be just to pickle and save additional results as "additional_spot_finder_results.pkl"